KsApi/queries/FetchProjectBySlugQuery.graphql

```diff
@@ -11,5 +11,6 @@ query FetchProjectBySlug($slug: String!, $withStoredCards: Boolean!) {
       id
       kind
     }
+    isPledgeOverTimeAllowed
```

Comment:
Do we not want to just add this to the project fragment? I think it would make more sense to have it there